src/app/info/components/InfoHero.tsx

```diff
@@ -74,21 +74,11 @@ const Card = ({
       href={link}
       className={`flex flex-col sm:w-fit grow-0 p-1.5 bg-neutral border border-line rounded-[6px] shadow-[0px_3.044px_9.131px_0px_rgba(0,0,0,0.02),0px_1.522px_1.522px_0px_rgba(0,0,0,0.03)]} hover:rotate-0 transition-all hover:z-10 hover:scale-110 ${className}`}
     >
-      <div
```

Comment:
Reverted this, this was added so that one of the images aligns, we have it sorted in image now